src/screens/journeysDisplay.tsx

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
What's goling on here? Are these top two lines needed?

Reply:
it made lint not angry

Reply:
What was the error this produced again?